cmake/Modules/FindComputeCpp.cmake

```diff
@@ -399,6 +403,8 @@ function(add_sycl_to_target)
     ${ARGN}
   )
 
+  set_target_properties(${SDK_ADD_SYCL_TARGET} PROPERTIES LINKER_LANGUAGE CXX)
```

Comment:
Nit: Obscure changes like this should probably have some explanation in the commit.